src/main/external-resources/renderers/Samsung-9series.conf

```diff
@@ -30,11 +31,11 @@ LoadingPriority = 1
 
 TranscodeVideo = MPEGTS-H264-AC3
 TranscodeAudio = WAV
-#MaxVideoWidth = 3840
-#MaxVideoHeight = 2160
+MaxVideoWidth = 3840
```

Comment:
Yes, it was tested by user, and recently many Samsung users on the forum ask for 4K, from series 6 to 9